pkg/runner/testdata/commands-waiting-for-input-freeze/push.yml

```diff
@@ -0,0 +1,13 @@
+name: freeze
+on: push
+
+jobs:
+  freeze:
+    runs-on: ubuntu-latest
+    steps:
+    - run: node # This command starts interactive and is waiting input
+    - run: git --version
+    - run: git clone https://github.com/nektos/act2 # This repo doesn't exist, trigger password prompt
+      continue-on-error: true
+      id: checkout
+    - run: exit ${{ tojson(steps.checkout.outcome) != '"failure"' && '1' || '0' }} # check if git actually failed
```

Comment:
This can be updated since https://github.com/nektos/act/pull/1136 is merged.

Reply:
Nevermind this comment. I read too fast and thought you're using the `1` and `0` to check against the outcome. 🤦